File: app/backend/models/portfolio.js
const {mongoose} =  require('../db');  // The mongodb connector library

let Portfolio = mongoose.model('Portfolio', {

  userId: {
    type: String,
    require: true
  },

  title:{
    type: String,
    require: true,
    unique: true
  },

  definition:{
    type: String,
    require: true
  },

  date:{
    type: Date,
    require: true
  },

  isPrivate: {
    type: Boolean,
    require: true
  }
});

module.exports = {
    Portfolio: Portfolio
}

File: app/android/app/src/main/java/com/example/arken/model/Portfolio.kt
package com.example.arken.model

import java.io.Serializable

class Portfolio(
    val _id: String?,
    val title:String,
    val definition: String,
    val isPrivate: Boolean,
    var tradingEqs: List<String>?,
    var userId: String?,
    var username: String?,
    var surname:String?
) : Serializable